public/pages/CreateTrigger/components/AddTriggerButton/AddTriggerButton.js

```diff
@@ -8,17 +8,21 @@ import _ from 'lodash';
 import { EuiButton } from '@elastic/eui';
 import { FORMIK_INITIAL_TRIGGER_VALUES } from '../../containers/CreateTrigger/utils/constants';
 
-const AddTriggerButton = ({ arrayHelpers, disabled }) => {
+const AddTriggerButton = ({
+  arrayHelpers,
+  disabled,
+  script = FORMIK_INITIAL_TRIGGER_VALUES.script,
```

Comment:
Curious about how the `script` parameter is used. Is this something specific to Cluster Metrics monitor?

Reply:
Each of the cluster metrics request types has it's own example script. This parameter allows us to pass in the request type-specific script using `clusterMetricsMonitorHelpers::getDefaultScript`.

Reply:
For now, it's really only used for a ClusterMetrics monitor, but it could be helpful for other applications in the future should we ever want to override the script in some way.